app/views/dashboard_v2/_sidebar.html.erb

```diff
@@ -55,7 +55,20 @@
     </div>
   </div>
 </div>
-
+<div>
```

Comment:
could we move this into its own partial - to load with `<%= render partial: "nodes/random" %>` and the partial template would be `_random.html.erb`.